src/bundle/Resources/public/js/scripts/fieldType/base/base-rich-text.js

```diff
@@ -220,6 +222,44 @@
 
             return alloyEditor;
         }
+
+        countWordsCharacters(container, editorHtml) {
+            let counterWrapper = container.parentElement.getElementsByClassName('ez-character-counter');
+
+            if (counterWrapper.length) {
+                let wordCharacterWrappers = counterWrapper[0].getElementsByTagName('span');
+                let characterCount = 0;
+                let wordCount = 0;
+                let countableTags = this.fetchCountableTags(editorHtml);
+
+                countableTags.forEach((tag) => {
+                    let sanitizedText = this.cleanWhiteCharacters(tag.innerText);
+
+                    wordCount += sanitizedText ? sanitizedText.split(' ').length : 0;
+                    characterCount += sanitizedText.length;
+                });
+
+                wordCharacterWrappers[0].innerText = wordCount;
+                wordCharacterWrappers[1].innerText = characterCount;
+            }
+        }
+
+        fetchCountableTags(html) {
+            const allowedTags = ['p:not(.ez-embed-content)', 'li', 'h1', 'h2', 'h3', 'h4', 'h5' , 'h6', 'th', 'td'];
+            const notCustomTagSelector = ':not([data-ezelement=ezattributes]) > ';
+
+            let allowedSelectors = allowedTags.map((item) => {
+                return notCustomTagSelector.concat(item);
+            });
+
+            return new DOMParser()
```

Comment:
If you will pass `doc` you don't need to do this parser. You can do querySelector on `doc`